WordPress/gutenberg
# Fix vertical overflow when inserter is open in post and site editor

## What?
Addresses #56811

**This needs further cross browser testing, but addresses the initial issue in Chrome desktop.**

## How?
Adding a class to the `InterfaceSkeleton` component in the site editor and post editor: `is-inserter-opened`. Target the class with `overflow: hidden`.

## Testing Instructions
1. Open the site editor (not sure if it happens in the post editor)
2. Open the inserter
3. Start typing and verify there is no odd vertical scrolling for the entire canvas, including the main canvas
